ydb/core/tx/scheme_board/helpers.h

```diff
@@ -65,5 +65,22 @@ TString JsonFromDescribeSchemeResult(const TString& serialized);
 
 bool ShouldIgnore(const TStateStorageInfo::TRingGroup& ringGroup);
 
+struct TClusterState {
+    ui64 Generation = 0;
+    ui64 Guid = 0;
+
+    TClusterState() = default;
+    explicit TClusterState(const TStateStorageInfo* info);
+    explicit TClusterState(const NKikimrSchemeBoard::TClusterState& proto);
+    NKikimrSchemeBoard::TClusterState ToProto() const;
```

Comment:
Обычно мы пишем так:
```suggestion
    void ToProto(NKikimrSchemeBoard::TClusterState& proto) const;
```
Это позволяет:
- лишний раз не включать описание протобуфов в заголовочные файлы;
- заполнять протобуф "по месту": `foo.ToProto(*proto.MutableFoo())` без лишних копирований и перемещений.